### PR TITLE
Print out a stack trace for the event if there's a throwable in there

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by Horneth** [comment]
:+1:  -> @cjllanwarne

File: src/main/scala/cromwell/logging/CromwellLogger.scala [hunk 1]
```diff
@@ -22,8 +26,18 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
      */
     val prefix = if (event.getMessage == "{}") s"[${TerminalUtil.highlight(129, "*")}] " else ""
 
-    s"$prefix[$level] $highlightedMessage\n"
+    s"$prefix[$level] $highlightedMessage\n${event.toStackTrace}"
```
Comment (mcovarr) [inline src/main/scala/cromwell/logging/CromwellLogger.scala]: This is well-behaved in the absence of a stack trace I assume?

Comment (geoffjentry) [inline src/main/scala/cromwell/logging/CromwellLogger.scala]: yeah

File: src/main/scala/cromwell/logging/CromwellLogger.scala [hunk 2]
```diff
@@ -22,8 +26,18 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
...
+object TerminalLayout {
+  val Converter = new ThrowableProxyConverter
+  Converter.start()
+
+  implicit class EnhancedILoggingEvent(val event: ILoggingEvent) extends AnyVal {
+    def toStackTrace: String = Converter.convert(event)
   }
 }
 
 class TerminalAppender extends ConsoleAppender
```
Comment (cjllanwarne) [inline src/main/scala/cromwell/logging/CromwellLogger.scala]: Is this no longer used?

**Comment by cjllanwarne** [comment]
:+1:

File: src/main/scala/cromwell/logging/CromwellLogger.scala (hunk 2 above)
Comment (geoffjentry) [inline src/main/scala/cromwell/logging/CromwellLogger.scala]: Honestly I assumed it was part of the logback magic. Let me see what happens if I remove it.

**Pull request merged**